doc/radosgw/lua-scripting.rst

```diff
@@ -295,6 +300,21 @@ Operations Log
 ~~~~~~~~~~~~~~
 The ``Request.Log()`` function prints the requests into the operations log. This function has no parameters. It returns 0 for success and an error code if it fails.
 
+Background Context
+--------------------
+The background context may be used for various need such as analytics, monitoring and
+caching data from other context executions.
+
+The ``RGW`` Lua table which is accesible from any context saves the data written into it
+while execution and this data could be read and used later in any other execution.
```

Comment:
maybe worth noting that in a cluster of several radosgws, each instance have its own separate `RGW` table, and the 'background' scripts will run on every instance